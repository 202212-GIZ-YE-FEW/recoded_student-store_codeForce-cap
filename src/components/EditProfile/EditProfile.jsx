export default function EditProfile() {
  return <div>EditProfile</div>
}
