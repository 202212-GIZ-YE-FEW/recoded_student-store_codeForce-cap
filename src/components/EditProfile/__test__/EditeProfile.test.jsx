import renderer from "react-test-renderer"

import EditProfile from "../EditProfile"

it("renders correctly", () => {
  const tree = renderer.create(<EditProfile />).toJSON()
  expect(tree).toMatchSnapshot()
})
