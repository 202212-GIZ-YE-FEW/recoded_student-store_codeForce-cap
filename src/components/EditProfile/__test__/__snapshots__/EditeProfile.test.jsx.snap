// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="Toastify"
  />
  <div
    className="grid lg:grid-cols-2 lg:ml-36 w-[86%] overflow-y-auto gap-x-12 mt-10 lg:mt-28 mx-auto h-[577px] md:h-[744px] lg:h-[100%]"
  >
    <label
      className="block lg:hidden cursor-pointer rounded-full"
      htmlFor="profileImg"
    >
      <img
        alt="..."
        className="rounded-full mx-auto mb-10"
        data-nimg="1"
        decoding="async"
        height={275}
        loading="lazy"
        onError={[Function]}
        onLoad={[Function]}
        src="/images/cat-photo.svg"
        style={
          {
            "color": "transparent",
          }
        }
        width={274}
      />
      <input
        accept="image/*"
        autoComplete="off"
        className="my-2 block w-full rounded-lg border border-gray-300 p-2.5 text-center hidden"
        id="profileImg"
        name="profileImg"
        onChange={[Function]}
        required={false}
        type="file"
      />
    </label>
    <span
      className="lg:h-[448px] md:h-[390px] flex flex-col justify-between "
    >
      <input
        autoComplete="off"
        className="my-2 block w-full rounded-lg border border-gray-300 p-2.5 text-center "
        maxLength={50}
        minLength={2}
        name="name"
        onChange={[Function]}
        placeholder="name"
        required={true}
        type="text"
      />
      <input
        autoComplete="off"
        className="my-2 block w-full rounded-lg border border-gray-300 p-2.5 text-center "
        maxLength={50}
        minLength={2}
        name="surname"
        onChange={[Function]}
        placeholder="surname"
        required={true}
        type="text"
        value=""
      />
      <input
        autoComplete="off"
        className="my-2 block w-full rounded-lg border border-gray-300 p-2.5 text-center "
        maxLength={50}
        name="email"
        onChange={[Function]}
        pattern="[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}"
        placeholder="email"
        required={true}
        type="email"
        value=""
      />
      <div
        className=" react-tel-input "
        onKeyDown={[Function]}
        style={{}}
      >
        <div
          className="special-label"
        >
          Phone
        </div>
        <input
          className="form-control "
          disabled={false}
          onBlur={[Function]}
          onChange={[Function]}
          onClick={[Function]}
          onCopy={[Function]}
          onDoubleClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          placeholder="1 (702) 123-4567"
          style={
            {
              "height": "50px",
              "width": "100%",
            }
          }
          type="tel"
          value="+967"
        />
        <div
          className="flag-dropdown "
          style={{}}
        >
          <div
            aria-haspopup="listbox"
            className="selected-flag"
            onClick={[Function]}
            role="button"
            tabIndex="0"
            title="Yemen: + 967"
          >
            <div
              className="flag ye"
            >
              <div
                className="arrow"
              />
            </div>
          </div>
        </div>
      </div>
      <input
        autoComplete="off"
        className="my-2 block w-full rounded-lg border border-gray-300 p-2.5 text-center "
        maxLength={50}
        minLength={8}
        name="newPassword"
        onChange={[Function]}
        placeholder="new-password"
        required={true}
        type="password"
        value=""
      />
      <input
        autoComplete="off"
        className="my-2 block w-full rounded-lg border border-gray-300 p-2.5 text-center "
        maxLength={50}
        minLength={8}
        name="confirmNewPassword"
        onChange={[Function]}
        placeholder="new-password-confirm"
        required={true}
        type="password"
        value=""
      />
      <input
        autoComplete="off"
        className="my-2 block w-full rounded-lg border border-gray-300 p-2.5 text-center lg:hidden block"
        name="address"
        onChange={[Function]}
        placeholder="address"
        required={true}
        type="text"
        value=""
      />
    </span>
    <span
      className="mt-3 hidden lg:block"
    />
    <span
      className="mx-auto mt-8"
    >
      <button
        className="h-12 w-56 lg:h-14 lg:w-64 m-1 p-1 px-2 hover:bg-purple hover:shadow-lg rounded-lg border-2 bg-pumpkin font-md text-[17px] text-white shadow-sm"
        type="submit"
      >
        save-changes
      </button>
    </span>
  </div>
</form>
`;
